dev/release/release-tag.sh

```diff
@@ -84,15 +84,24 @@ sed -i".tmp1" 's/version = .*$/version = '"$RELEASE_VERSION"'/g' gradle.properti
 sed -i".tmp2" 's/    version=.*$/    version="'"$PYGRAVITINO_RELEASE_VERSION"'",/g' clients/client-python/setup.py
 sed -i".tmp3" 's/^version = .*$/version = \"'"$RELEASE_VERSION"'\"/g' clients/filesystem-fuse/Cargo.toml
 sed -i".tmp4" 's/^appVersion: .*$/appVersion: '"$RELEASE_VERSION"'/g' dev/charts/gravitino/Chart.yaml
-sed -i".tmp5" '22s/  tag: .*$/  tag: '"$RELEASE_VERSION"'/g' dev/charts/gravitino/values.yaml
+sed -i".tmp5" '34s/  tag: .*$/  tag: '"$RELEASE_VERSION"'/g' dev/charts/gravitino/values.yaml
```

Comment:
Do we need a check logic here? If others modify this file, the number may change.